services/amazon_streaming.yml

```diff
@@ -0,0 +1,35 @@
+id: amazon_streaming
+name: Amazon Streaming
+rules:
+  - '||amazonprimevideo.cn^'
+  - '||amazonprimevideo.com.cn^'
+  - '||amazonprimevideos.com^'
+  - '||primevideo.cc^'
+  - '||primevideo.com^'
+  - '||primevideo.info^'
+  - '||primevideo.org^'
+  - '||primevideo.tv^'
+  - '||amazonvideo.cc^'
+  - '||amazonvideo.com^'
+  - '||amazonvideodirect.com^'
+  - '||prime-video.com^'
+  - '||music.amazon.com^'
+  - '||music.amazon.co.uk^'
+  - '||music.amazon.in^'
+  - '||amazonmusic.com^'
+  - '||music.a2z.com^'
+  - '||aiv-cdn.net^'
+  - '||aiv-delivery.net^'
+  - '||atv-ext.amazon.com^'
+  - '||atv-ext-eu.amazon.com^'
+  - '||atv-ext-fe.amazon.com^'
+  - '||atv-ps.amazon.com^'
+  - '||atv-ps-eu.amazon.com^'
+  - '||atv-ps-eu.amazon.co.uk^'
+  - '||atv-ps-fe.amazon.co.jp^'
+  - '||atv-ps-fe.amazon.com^'
+  - '||av-eu.amazon.com^'
+  - '||av-na.amazon.com^'
+  - '||pv-cdn.net^'
```

Comment:
Not sure about CDNs
aiv-cdn.net
pv-cdn.net
I am sure the rest of the domains are enough to block related services.